manifests/dnssec-interference.yml

```diff
@@ -0,0 +1,9 @@
+---    
+description: DNSSEC Interference Study
+repo-prefix: dnssecinterferencestudy
+active: true
+private-repo: false
+artifacts:
+    - web-ext-artifacts/dnssec-interference-study.xpi
+addon-type: privileged
```

Comment:
The `privileged` type is reserved for product-like extensions that user actively and willingly install through an interaction (usually clicking an "Install" button on a website).
Add-ons of that type also apply to [Mozilla's add-on policies](https://extensionworkshop.com/documentation/publish/add-on-policies/) and need to go through the respective review process to ensure those policies are met. Additionally, privileged parts of the code need to go through Security review.